asmx/src/org/objectweb/asmx/ClassReader.java

```diff
@@ -1480,14 +1480,18 @@ private int readAnnotationValues(
     *        {@link #readConst readConst}.
     * @param mv the visitor to generate the visitor that must visit the values.
     * @param visible {@code true} if the annotation is visible at runtime.
+    * @param is_method_extended_annotations {@code true} if this method is
```

Comment:
Use Java-standard camel-case naming: `isMethodBody` seems appropriate, as the method is always about annotations.